namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-incentives-prod/resources/rds.tf

```diff
@@ -9,9 +9,14 @@ module "dps_rds" {
   environment-name            = var.environment
   infrastructure-support      = var.infrastructure_support
   rds_family                  = var.rds_family
-  allow_major_version_upgrade = "false"
-  db_instance_class           = "db.t3.small"
+  db_instance_class           = "db.t3.large"
```

Comment:
Igor suggests we may well use the [newer instances](https://aws.amazon.com/rds/instance-types/) `db.t4g.large`.

Reply:
…because it's a bit better performant and cheaper. plus, it's likely to have a longer pg version upgrade path

Reply:
FYI: Done here: https://github.com/ministryofjustice/cloud-platform-environments/pull/10462/commits/4a573f9b79987523e8fa1e9fff2b46f3bf81d0ba